pkg/operators/common/kubeinventorycache.go

```diff
@@ -49,24 +55,19 @@ var (
 
 func GetK8sInventoryCache() (*K8sInventoryCache, error) {
 	once.Do(func() {
-		cache, err = newCache(10 * time.Minute)
+		cache, err = newCache()
 	})
 	return cache, err
 }
 
-func newCache(defaultResync time.Duration) (*K8sInventoryCache, error) {
+func newCache() (*K8sInventoryCache, error) {
```

Comment:
I'm wondering if we really need this function anymore. At this point we can move the client set creation into the `cache.useCount == 0` conditional and just return `&K8sInventoryCache{}` on `GetK8sInventoryCache`.

Reply:
I left the client set creation here so it will be done only once, because it is really only needed once

Reply:
But the creation is not very expensive, right? Perhaps it's more expensive to keep it on memory than creating it each time it's needed. However it's not a big blocker for me.